src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySink.java

```diff
@@ -310,4 +329,51 @@ protected Configuration getOutputConfiguration() throws IOException {
     return configuration;
   }
 
+  public static void validateJsonStringFields(Schema schema,
```

Comment:
Add javadoc and if method is not getting used outside, make it private or protected based on usage

Reply:
```java
/**
 * Validates that the fields to be converted to JSON strings are present in the Output Schema.
 * @param schema Output Schema.
 * @param jsonStringFields List of fields to be converted to JSON strings comma separated.
 * @param collector FailureCollector to collect errors.
 */
```